package.json

```diff
@@ -24,7 +25,7 @@
     "web-vitals": "^2.1.0"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "set \"GENERATE_SOURCEMAP=false\" && react-scripts start",
```

Comment:
What is GENERATE_SOURCEMAP for? Is this the right place to set the variable?

Reply:
It is used to remove a warning that I get in my console when I start the project. When I start the react app it generated the source map files so some packages are causing an issue of which I have added a screenshot. When we do the above solution it works without any warning in the console.

![image](https://user-images.githubusercontent.com/85132632/215523968-e5a453c9-fbab-4366-a08d-220bd1589214.png)